.github/workflows/nightly.yml

```diff
@@ -0,0 +1,31 @@
+name: Nightly Release
+
+on:
+  schedule:
+    - cron:  '0 9 * * *'
```

Comment:
Does it make sense to release in the case there were no releases?

Reply:
If there weren't changes in a day, seems to me like `twine upload` is gonna fail along with the whole workflow.

Reply:
Oh good point, thanks - i'll figure that out :)